packages/cli/src/commands/database/seed/tables.ts

```diff
@@ -150,6 +151,12 @@ export const seedTables = async (
   await seedOidcConfigs(connection, defaultTenantId);
   await seedAdminData(connection, defaultManagementApi);
 
+  /**
+   * Create a pre-configured role for the Logto Management API access
+   * in the default tenant (the default tenant is the only tenant for the OSS version, and the initial tenant for cloud).
+   */
+  await seedPreConfiguredManagementApiAccessRole(connection, defaultTenantId);
```

Comment:
How about putting the logic in the seedAdminData(connection, defaultManagementApi)

Reply:
seedAdminData will also be used for the admin tenant, which is not expected